### ISSUE TITLE
Use the term "members" instead of "replicas" for stateful set pods

### ISSUE THREAD

**Issue by the-gigi** [kubernetes/website#2103]
**This is a...** 

- [x] Feature Request
- [ ] Bug Report

**Problem:**

Stateful sets use the term "replica" to refer to members with unique identifiers in the set. This term is confusing and inaccurate because the members are by definition unique and in general don't replicate the same state. The term is also overloaded twice:

1. For stateless pods "replicas" are indeed the correct term because all stateless pods are cut from the spec are equivalent
2. For many data stores "replicas" means nodes that contain the same subset of the data replicated for high-availability and redundancy.

**Proposed Solution:**

Use the term "members" instead of "replicas" to refer to the pods of a stateful set. 

If this is accepted then probably the spec filed name should be updated too.

**Page to Update:**
http://kubernetes.io/...

**Comment by jaredbhatti** [comment]
Need someone to verify this language.

**Comment by sftim** [comment]
I think it's OK to:
/priority backlog

**Comment by sftim** [comment]
My view is that the rename should happen first in the StatefulSet implementation itself, and then the docs can follow.
